test/extended/cli/builds.go

```diff
@@ -204,9 +204,7 @@ var _ = g.Describe("[sig-cli] oc builds", func() {
 
 		g.JustBeforeEach(func() {
 			g.By("getting the api server host")
-			err := oc.Run("project").Args("default").Execute()
-			o.Expect(err).NotTo(o.HaveOccurred())
-			out, err := oc.WithoutNamespace().Run("status").Args().Output()
+			out, err := oc.WithoutNamespace().Run("--namespace=default", "status").Args().Output()
```

Comment:
the `oc project default` switches the project which impacts the process and create later on.  Which namespace are those supposed to be in?

Reply:
```
oc project default
oc status
```
This sequence of commands was being used to get the url of the api server, in the
majority of cases "oc project default" is most likely a NOOP with the negative 
side-effect of leaving KUBECONFIG as a empty file for a brief
period of time (causing other tests to fail).  

```
oc --namespace=default status
```
does the same thing without writing to KUBECONFIG so doesn't effect other tests

Reply:
> the oc project default switches the project which impacts the process and create later on. Which namespace are those supposed to be in?

The process and create commands are being run with a namespace e.g.

Reply:
What this does can actually be done simpler with either:
- `oc whoami --show-server`
- `oc config view -o jsonpath='{.clusters[*].cluster}'`